package.json

```diff
@@ -43,7 +42,8 @@
     "@polymer/paper-ripple": "^3.0.2",
     "@polymer/paper-styles": "^3.0.1",
     "@polymer/paper-toast": "^3.0.1",
-    "@sentry/electron": "^3.0.7",
```

Comment:
I doubt an upgrade will fix the problem. In fact, perhaps an upgrade broke it. (dependabot?)
Did you check https://github.com/getsentry/sentry-electron/blob/master/MIGRATION.md ?

/cc @jyyi1

Reply:
I'm not saying we shouldn't upgrade, but it would be helpful to understand what the problem actually is and if we are using the APIs correctly.